linuxkm/lkcapi_glue.c

```diff
@@ -245,18 +595,21 @@ static void km_AesExit(struct crypto_skcipher *tfm)
     km_AesExitCommon(ctx);
 }
 
-#endif /* LINUXKM_LKCAPI_REGISTER_ALL ||
-        * LINUXKM_LKCAPI_REGISTER_AESCBC ||
-        * LINUXKM_LKCAPI_REGISTER_AESCFB
+#endif /* LINUXKM_LKCAPI_REGISTER_AESCBC ||
```

Comment:
Could it be worth having a master `LINUXKM_LKCAPI_REGISTER_AES` that is || of them all?

Then a separate one for AES non-aead?

Reply:
Hey that's a great idea.  Will fix.